pyro/contrib/epidemiology/compartmental.py

```diff
@@ -253,6 +267,20 @@ def transition(self, params, state, t):
         """
         raise NotImplementedError
 
+    def finalize(self, params, state):
+        """
+        Optional method for likelihoods that depend on the entire time series.
```

Comment:
Clarify here that this is only for non-factorized likelihoods and should be avoided if possible in favor of `transition`?

Reply:
done